js/main.js

```diff
@@ -14,39 +19,33 @@ const getRandomInt = (min = 0, max = 0) => ((min >= 0) && (min < max)) ? Math.fl
 //  Проверяет длину коментария
 const checkCommentLength = (str = '', maxCommentLength = 140) => str.length <= maxCommentLength;
 
-getRandomInt();
-checkCommentLength();
-
-// Получает не повторяющееся число из диапазона
-const getRandomNonRepeatingNumbers = (min, max) => {
-  const previousValues = [];
-  let currentValue = getRandomInt(min, max);
-  if (previousValues.length >= (max - min + 1)) {
-    throw new Error(`Перебраны все числа из диапазона от ${  min  } до ${  max}`);
-  }
-  while (previousValues.includes(currentValue)) {
-    currentValue = getRandomNumber(min, max);
+const getUnquiArray = () => {
+  const result = new Set;
+  while (result.size < 25) {
+    result.add(getRandomInt(1,300));
   }
-  previousValues.push(currentValue);
-  return currentValue;
+  return Array.from(result);
 };
 
 const getArrayMocks = () => {
-  for (let i = 1; i <= MAXPOSTS; i++) {
+  const result = [];
+  const moksCommentsArray = [];
+  const UNIQUI_ELEMENTS = getUnquiArray();
+
+  for (let index = 0; index < MAX_POST; index++) {
     moksCommentsArray.push({
-      id: getRandomNonRepeatingNumbers(1,300),
-      avatar: 'img/avatar-' + getRandomInt(0, 6) + '.svg',
+      id: UNIQUI_ELEMENTS[index],
+      avatar: AVATAR_SRC + getRandomInt(0, 6) + AVATAR_FORMAT,
       message: MESSAGES[getRandomInt(0, MESSAGES.length - 1)],
       name: COMMENTS_NAMES[getRandomInt(0, COMMENTS_NAMES.length - 1)],
-    })
-    moksPostsArray.push({
-      id: i,
-      url: 'photos/' + i + '.jpg',
+    });
+    result.push({
+      id: index + 1,
+      url: PHOTO_SRC + index + 1 + PHOTO_FORMAT,
```

Comment:
```suggestion
      url: PHOTO_SRC + (index + 1) + PHOTO_FORMAT,
```

А еще лучше вопспользовать шаблонными строками:
```suggestion
      url: `${PHOTO_SRC}${index + 1}${PHOTO_FORMAT}`,
```